recipes/mastercard/runner.sh

```diff
@@ -72,7 +90,8 @@ VERSION=$DATE
     )
     
     #unsplit csv is too large. Must compress.
-    zip -9 output/mastercard_$DATE.zip output/mastercard.csv
+    echo "compressing mastercard.csv"
+    zip -9 output/daily_transactions.zip output/mastercard.csv
```

Comment:
does it need to be zipped in "output" folder?

Reply:
The mastercard csv is too large to upload. It must be compressed, and the raw csv deleted.

Reply:
I was asking more about zipping it in the output folder vs. zipping the file.